src/UXClient/styles.scss

```diff
@@ -311,6 +311,10 @@ select::-ms-expand {
     }
 }
 
+.tsi-baseMono {
+    font-family: "Lucida Console", Monaco, monospace;
```

Comment:
What about something like this???

![image](https://user-images.githubusercontent.com/4854357/90682140-62ed4c80-e219-11ea-822b-7206110c44d7.png)

Reply:
background themed obviously